tests/e2e/nginx-encrypted.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nginx
+  name: nginx-encrypted
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+      annotations:
+        io.containerd.cri.runtime-handler: RUNTIMECLASS
+    spec:
+      runtimeClassName: RUNTIMECLASS
+      containers:
+        - image: docker://ghcr.io/confidential/nginx:latest
```

Comment:
The name of the image can change depending of the secrets that we will have for the GHA

Reply:
Same comment as above

Reply:
changes applied